### ISSUE TITLE
force close remote-notifier 0.2.6

### ISSUE THREAD

**Issue by GoogleCodeExporter** [CarlitoBriganteNT/android-notifier#157]
```
IMPORTANT: PLEASE search the existing issues (including closed ones) for
something similar before filing a new one.

What steps will reproduce the problem?
1. pair phone and computer - authorize phone to access any bluetooth services 
on computer
2. open android-desktop-notifier 
3. send test notification in remote-notifier on phone
4. force close occurs

What is the expected output? What do you see instead?
expected to see test notification sent
remote-notifier force closes

What version of the product are you using? On what operating system?
Samsung Galaxy S with Froyo I9000XXJPC rom
Windows XP SP3 32-bit & android-notifier-desktop 0.4.0
Growl for Windows v2.0.4

attached are logcat from Android phone


Please provide any additional information below.

```

Original issue reported on code.google.com by `luke.a.i...@gmail.com` on 20 Sep 2010 at 12:43

Attachments:
- [2010-09-20-10-32-49.txt](https://storage.googleapis.com/google-code-attachments/android-notifier/issue-157/comment-0/2010-09-20-10-32-49.txt)
- [2010-09-20-10-33-04.txt](https://storage.googleapis.com/google-code-attachments/android-notifier/issue-157/comment-0/2010-09-20-10-33-04.txt)
- [2010-09-20-10-35-55.txt](https://storage.googleapis.com/google-code-attachments/android-notifier/issue-157/comment-0/2010-09-20-10-35-55.txt)

**Comment by GoogleCodeExporter** [comment]
```
It's not clear - is the force close on the android app or on the desktop app?
```

Original comment by `rdamazio@gmail.com` on 20 Sep 2010 at 7:38
- Changed state: **Waiting**

**Comment by GoogleCodeExporter** [comment]
```
Ok, from the logs it looks like it's the Android app, sorry about that.
Indeed I see other reports on the android market:
java.lang.NullPointerException
at 
org.damazio.notifier.notification.BluetoothNotificationMethod.doSendNotification
(BluetoothNotificationMethod.java:154)
at 
org.damazio.notifier.notification.BluetoothNotificationMethod.sendNotification(B
luetoothNotificationMethod.java:95)
at 
org.damazio.notifier.notification.Notifier.runNotificationThread(Notifier.java:6
9)
at org.damazio.notifier.notification.Notifier.access$0(Notifier.java:65)
at org.damazio.notifier.notification.Notifier$1.run(Notifier.java:54)


```

Original comment by `rdamazio@gmail.com` on 20 Sep 2010 at 7:41
- Changed state: **Accepted**
- Added labels: **Component-Logic**, **OpSys-Android**, **Priority-High**, **Stability**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Found it :)
```

Original comment by `rdamazio@gmail.com` on 20 Sep 2010 at 7:44
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in r268. Thanks for reporting.
```

Original comment by `rdamazio@gmail.com` on 20 Sep 2010 at 7:47
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
yep - i fixed the issue by uninstalling 0.2.5 and then installing 0.2.6

must be something up with the upgrade

is r268 available in the market?
```

Original comment by `luke.a.i...@gmail.com` on 20 Sep 2010 at 9:16

**Comment by GoogleCodeExporter** [comment]
```
Not yet - it will be in the next version.
```

Original comment by `rdamazio@gmail.com` on 20 Sep 2010 at 2:08
